src/index.html

```diff
@@ -6,7 +6,47 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar">
+      <div class="
+          calendar__container
+          calendar__container--start-day--sun
+          calendar__container--month-length--29
```

Comment:
```suggestion
    <div class="calendar calendar--start-day--sun calendar--month-length--29">
```
You can simplify, you don't need an additional wrapper in this case